src/Altinn.Broker.Application/GetFileOverviewQuery/GetFileOverviewQueryHandler.cs

```diff
@@ -34,7 +34,16 @@ public GetFileOverviewQueryHandler(IResourceOwnerRepository resourceOwnerReposit
         {
             return Errors.FileNotFound;
         }
-        var hasAccess = await _resourceRightsRepository.CheckUserAccess(file.ResourceId, request.Token.ClientId, ResourceAccessLevel.Write);
+        var hasAccess = false;
```

Comment:
Is there any way to do this test so that we do not have to add custom code in every application handler? For instance, by adding it in _resourceRightsRepository.CheckUserAccess instead. That way, we'll have isolated the logic to one place instead of ~8.

Reply:
Agreed, moved into ResourceRightsRepository so as to centralize logic.